src/doc_builder/external.py

```diff
@@ -54,14 +54,17 @@ def post_process_objects_inv(object_data, doc_url):
     return links
 
 
-def get_stable_version(package_name):
+def get_stable_version(package_name, repo_owner="huggingface", repo_name=None):
     """
     Gets the version of the last release of a package.
 
     Args:
         package_name (`str`): The name of the package.
+        repo_owner (`str`): The owner of the GitHub repo.
+        repo_name (`str`): The name of the GitHub repo. If not provided, will be the same as the package name.
     """
-    github_url = f"https://github.com/huggingface/{package_name}"
+    repo_name = repo_name or package_name
```

Comment:
Let's not rely on magic bool conversion :-)
```suggestion
    repo_name = repo_name if repo_name is not None else package_name
```